packages/types/src/providers/cerebras.ts

```diff
@@ -3,9 +3,18 @@ import type { ModelInfo } from "../model.js"
 // https://inference-docs.cerebras.ai/api-reference/chat-completions
 export type CerebrasModelId = keyof typeof cerebrasModels
 
-export const cerebrasDefaultModelId: CerebrasModelId = "qwen-3-coder-480b-free"
+export const cerebrasDefaultModelId: CerebrasModelId = "gpt-oss-120b"
 
 export const cerebrasModels = {
+	"zai-glm-4.6": {
+		maxTokens: 40000,
```

Comment:
This seems like a lot if the context window is only 128k. I'd recommend going with 16,384 like similar models, even if it's theoretically possible for the model to do more.
```suggestion
		maxTokens: 16_384,
```